packages/web3-providers/src/DSearch/index.ts

```diff
@@ -385,32 +385,42 @@ export class DSearchAPI<ChainId = Web3Helper.ChainIdAll, SchemaType = Web3Helper
     private async searchCollectionListByTwitterHandler(
         twitterHandler: string,
     ): Promise<Array<SearchResult<ChainId, SchemaType>>> {
-        const collections = (
-            await Promise.allSettled([
-                this.CoinGeckoClient.get(),
-                this.CoinMarketCapClient.get(),
-                this.NFTScanCollectionClient.get(),
-            ])
+        const collections = uniqWith(
+            (
+                await Promise.allSettled([
+                    this.CoinGeckoClient.get(),
+                    this.CoinMarketCapClient.get(),
+                    this.NFTScanCollectionClient.get(),
+                ])
+            )
+                .map((v) => (v.status === 'fulfilled' && v.value ? v.value : []))
```

Comment:
flatMap

Reply:
Fixed.
Casting because `Array<FungibleTokenResult<ChainId, SchemaType> | Array<NonFungibleCollectionResult<ChainId, SchemaType>` is not assignable to `Array<FungibleTokenResult<ChainId, SchemaType> | NonFungibleCollectionResult<ChainId, SchemaType>>`

Reply:
A bug of TS. 😢